src/test/java/template/pageobjects/pages/HomePage.java

```diff
@@ -21,9 +19,10 @@
      */
     @Override
     @Step("ensure this is a homepage")
-    public void isExpectedPage()
+    public HomePage isExpectedPage()
     {
         $("#service-areas").should(exist);
```

Comment:
please call `super.isExpectedPage();` just in case we would add some default functionality.

Reply:
Done

Reply:
still can't see the call of super.isExpectedPage

Reply:
as `isExpectedPage` method of `AbstractPageObject` is now abstract, it's not possible to call it here because the `HomePage` is a direct child of `AbstractPageObject`

Reply:
I see ^^ My bad, sorry